configure.ac

```diff
@@ -89,6 +89,12 @@ if test "$enable_gdb_debug" = "yes"; then
 	AC_DEFINE([CONFIG_GDB_DEBUG], [1], [Enable debugging with GDB])
 fi
 
+# check if we should enable heap alloc debugging
+AC_ARG_ENABLE(gdb_debug, [AS_HELP_STRING([--enable-heap-debug],[heap debug supported])], enable_heap_debug=$enableval, enable_heap_debug=no)
```

Comment:
```suggestion
AC_ARG_ENABLE(heap_debug, [AS_HELP_STRING([--enable-heap-debug],[heap debug supported])], enable_heap_debug=$enableval, enable_heap_debug=no)
```

Reply:
yep, copy and paste from the gdb config....